src/ui.rs

```diff
@@ -26,9 +27,18 @@ struct Stage {
     expand: bool,
 }
 
+/// Returns whether the terminal is attended or not
+fn is_attended(term: &Term) -> bool {
```

Comment:
nit: this is kind of abusing the meaning of attended. Shouldn't we have a function to decide if we strip  styling based on where it is attended and the env variable is not set.

Reply:
Good point. Will rename to something more appropriate